### DISCUSSION TITLE
How About a Key to Use RefindPlus with Secure Boot?

### DISCUSSION THREAD

**Discussion by 1457384613gh** [dakanji/RefindPlus#65] [General]
To load RefindPlus after enabled secure boot, could you give a secure boot key ?
It cannot load RefindPlus unless disabled secure boot.

**Comment by dakanji** [comment]
You might want to consider using rEFInd instead if you require secure boot.
Haven't looked into this on RefindPlus.

**Reply by volcbs** [reply]
To be perfectly fair, win11 will require secure boot to be used. So if the users want to dual boot with the new win11, it'll be difficult if they want to use RefindPlus.

**Reply by dakanji** [reply]
Are you confirming you have tried this feature after taking all the required user steps and that it doesn't work?

**Reply by volcbs** [reply]
Nope, I just explain about what he meant by that having secure boot feature in RefindPlus. It's probably related to that.

**Reply by dakanji** [reply]
I see. I understood what he was talking about which was why I referred him to an alternative that might work for him

**Reply by volcbs** [reply]
Well, this fork has "use_graphic_for" setting for opencore and clover while the original one doesn't after all, which means it supposedly supports hackintosh more than the original software regardless what it is originally intended. I can understand his sentiment about requesting that to you.

Probably your reasoning to reject the idea is more heavily toward MacOS that doesn't really use secureboot? or something along the way that prevents the old iMac to boot MacOS?

**Reply by dakanji** [reply]
I am not sure where you have seen a rejection. I simply said I have not gotten round to look at this aspect.

A report does not mean it is indeed the case that something is not working as it may simply be down to misconfiguration or not following the usage instructions. You recently reported that default_selection was not working yourself as an example.

Bear in mind that you haven't actually confirmed that it doesn't work yourself. If you want to make a positive contribution, please verify whether it works or not.

**Reply by volcbs** [reply]
Ah, fair enough. Sorry for taking your time here. Well, I might report to you when it happens. Don't think it'll be soon though. Thanks anyway.

**Reply by dakanji** [reply]
Feel free to ask questions, that's what this facility is for.

If you do test this item at some point and find problems with the implementation, raise an issue. 
There is a template to guide the provision of required background information.

**Comment by dakanji** [comment]
See this Discussion Item for how to use RefindPlus with Secure Boot: https://github.com/dakanji/RefindPlus/discussions/190
In terms of keys specifically, keys provided by the upstream project can be used.
